### PR TITLE
Fixes #671

### LINKED ISSUE #671: Replay directory is only created if it exists??!!!
ReploayServiceImpl.class line 158

### PR DESCRIPTION
**Pull request by axel1200**
See yourself

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ## Directory-based project format:
-#.idea/
+.idea/
```
Comment (micheljung) [inline .gitignore]: Don't ignore .idea, there are files in there that need to be shared

File: src/main/java/com/faforever/client/replay/ReplayServiceImpl.java [hunk 2]
```diff
@@ -154,7 +154,7 @@ static String guessModByFileName(String fileName) {
     String replayFileGlob = clientProperties.getReplay().getReplayFileGlob();
 
     Path replaysDirectory = preferencesService.getReplaysDirectory();
-    if (!Files.notExists(replaysDirectory)) {
+    if (Files.notExists(replaysDirectory)) {
```
Comment (micheljung) [inline src/main/java/com/faforever/client/replay/ReplayServiceImpl.java]: Aah nice

**Comment by micheljung** [comment]
Oh and, the commit message should say what it does w/o requiring the reader to look at github

**Comment by axel1200** [comment]
Made a propper commit message

**Pull request merged**